doc/module_examples/03_action_required/check.sh

```diff
@@ -0,0 +1,47 @@
+#!/bin/bash
+
+. /usr/share/preupgrade/common.sh
+
+#END GENERATED SECTION
+
+#
+# The whole section above is processed and modified by the preupg-xccdf-compose
+# script, according to the contents of the INI file (in this case it is the
+# content.ini file). In addition, the LICENSE used by the Preupgrade Assistant
+# is inserted.
+#
+
+##
+# Briefly:
+#   In case a manual inspection, check or action BEFORE the upgrade is needed,
+#   do the following things:
+#     1) provide a text in $SOLUTION_FILE: a description of the problem and
+#        remediation instructions
+#     2) use "log_high_risk" to provide a short message that a problem was
```

Comment:
Does it really need to be exactly "log_high_risk" if I need to request user to do something before the upgrade? I mean it in a sense that it can't be "log_medium_risk" for example.

- If **so**, reword this bullet to something like:
*2) use "log_[severity]_risk" to provide a short message that a problem was found. Change [severity] to one of the levels of severity available through API, for example "log_medium_risk"*

- If **not**, you can prevent others asking this question :) For example by adding something like *"Preupgrade Assistant requires the risk to be set to high in this scenario"*